app/src/main/java/com/example/calculator/utils/ErrorEnum.java

```diff
@@ -0,0 +1,7 @@
+package com.example.calculator.utils;
+
+public enum ErrorEnum {
+    INCOMPLETE_OPERATION_ERROR,
+    DIVIDE_BY_0_ERROR,
```

Comment:
do not use number on the names..change it by DIVIDE_BY_ZERO_ERROR

Reply:
Solved!